wpimath/src/main/java/edu/wpi/first/wpiutil/math/MatrixUtils.java

```diff
@@ -1,83 +0,0 @@
-/*----------------------------------------------------------------------------*/
```

Comment:
This is a breaking change... MatrixUtils.java was distributed to teams for the 2020 season.  Since this is a utility class, it seems like it should be easy to only deprecate it rather than remove it?

Reply:
Sure, I've deprecated it in be466c0